Sprint-1/fix/median.test.js

```diff
@@ -4,6 +4,28 @@
 // passing all the tests...
 // Fix the implementation of calculateMedian so it passes all tests
 
+/*const calculateMedian = require("./median.js");
+
+describe("calculateMedian", () => {
+  test("returns the median for odd length array", () => {
+    expect(calculateMedian([1, 2, 3])).toEqual(2);
+    expect(calculateMedian([1, 2, 3, 4, 5])).toEqual(3);
+  });
+
+  test("returns the average of middle values for even length array", () => {
+    expect(calculateMedian([1, 2, 3, 4])).toEqual(2.5);
+    expect(calculateMedian([1, 2, 3, 4, 5, 6])).toEqual(3.5);
+  });
+
```

Comment:
Your other committed files include a test with an empty array argument.  Can you include one here?  What do you want it to return?